client/src/lib/__mocks__/jquery.js

```diff
@@ -0,0 +1,34 @@
+const $ = jest.genMockFromModule('jquery');
+
+const ajax = (options) => {
+  const fakeData = {
+    inputOptions: options,
+    madeToday: 200,
+    reservations: [{
+      time: 17,
+      remaining: 1,
+    },
+    {
+      time: 18,
+      remaining: 2,
+    },
+    {
+      time: 19,
+      remaining: 3,
+    },
+    {
+      time: 20,
+      remaining: 4,
+    },
+    {
+      time: 21,
+      remaining: 5,
+    },
+    ],
+  };
+  options.success(fakeData);
+};
+
+$.ajax = ajax;
+
```

Comment:
Not completely sure what this file is for/does. Can you shed some light?

Reply:
So this is basically replacing the real $.ajax() with my fake $.ajax() for testing purposes.  I do it so that I can check what my helper function passes in to $.ajax without actually making a real AJAX request.  My fake $.ajax() then returns some fake data and I can run additional tests to make sure my client is processing the returned data correctly (not yet in the test file)